drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/analysis/cache/RuleInspectorDumper.java

```diff
@@ -68,6 +74,28 @@ private void dumpConditions() {
         dump.append( "}\n" );
     }
 
+    private void dumpActions() {
+
+        dump.append( "Actions{\n" );
+        for ( final ActionsInspectorMultiMap actionsInspectorMultiMap : ruleInspector.getActionsInspectors() ) {
+            dumpAction( actionsInspectorMultiMap );
+        }
+        dump.append( "}\n" );
+
+    }
+
+    private void dumpAction( final ActionsInspectorMultiMap actionsInspectorMultiMap ) {
+        for ( final Object object : actionsInspectorMultiMap.allValues() ) {
+            dump.append( "Action{\n" );
+            if ( object instanceof HumanReadable ) {
+                dump.append( (( HumanReadable ) object).toHumanReadableString() );
+            } else {
+                dump.append( object.toString() );
+            }
+            dump.append( "}\n" );
+        }
+    }
+
```

Comment:
Just my helper class for situations where GWT debugger does not fire up ( about 7 times out of 10 ) or just when I need a more readable version of the rule inspector.